src/generate-preload-links.js

```diff
@@ -4,7 +4,7 @@ export default function generatePreloadLinks(fontNames, fontDictionary) {
     const font = fontDictionary[fontName];
     if (font) {
       links.push(
-        `\n<link rel="font" href="/_static/${
+        `\n<link rel="font" href="${
           font.urls.woff2
```

Comment:
This is the line I am talking about. It's only doing woff2 here...

Reply:
We can't detect woff2 support on the server, but in the cases where it's
     not supported (IE 8, Safari <= El Capitan) this will silently fail and
     styles will trigger load a little later